tests/integration/defs/disaggregated/test_disaggregated.py

```
@@ -745,30 +756,119 @@ def setup_disagg_cluster(
+
+            done, pending = await asyncio.wait(
+                [ready_task, tick_task],
+                return_when=asyncio.FIRST_COMPLETED,
+            )
+
+            for t in pending:
+                t.cancel()
+                try:
+                    await t
+                except (asyncio.CancelledError, Exception):
+                    pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)